### PR TITLE
feat(webterminal-plugin): Add tests for webterminal plugin

### PR DESCRIPTION
**Pull request by mimotej**
SSIA

### CODE REVIEW

**Review by tumido — CHANGES_REQUESTED** [review]
Just a nit. Otherwise, LGTM 👍 This looks great! 🙂

File: plugins/web-terminal/src/components/TerminalComponent/mocks/createWorkspace.json
[the remark this answers is not in the document]
Comment (mimotej) [reply plugins/web-terminal/src/components/TerminalComponent/mocks/createWorkspace.json:1]: Done. :)

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**